hack/community/generate.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+# Generate the new community manifests, from the current WMCO manifests.
+
+# Must be ran within WMCO root.
+
+# Example:
+# Cut community v5.1.0 for community-4.10
+# Run: bash ./hack/community/generate.sh community-4.10
+
+clone_community() {
+  echo
+  echo "## Cloning community-operator repo"
+  git clone "${CO_FORK}" "${CO_DIR}"
+  pushd "${CO_DIR}"
```

Comment:
please pass these variables into the function instead of globally setting them elsewhere